src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -85,4 +93,40 @@ public static function getTimezones()
 
         return static::$timezones;
     }
+
+    /**
+     * Returns the timezone choices.
+     *
+     * The choices are generated from the ICU function
+     * \DateTimeZone::listIdentifiers(). They are cached during a single request,
+     * so multiple timezone fields on the same page don't lead to unnecessary
+     * overhead.
+     *
+     * @return array The timezone choices
+     */
+    private static function getFlippedTimezones()
+    {
+        if (null === static::$timezones) {
```

Comment:
this must use ``self::`` to access private properties, not ``static::``. Otherwise the code is broken when extending the class (as it would try to access the property on the child class, where it is not there)